dist/images/Dockerfile

```diff
@@ -41,7 +41,7 @@ ARG dpdkVer=18.11.2
 ARG dpdkSubVer=1.el7
 ARG dpdkRpmUrl=http://mirror.centos.org/centos/7/extras/${rpmArch}/Packages/dpdk-${dpdkVer}-${dpdkSubVer}.${rpmArch}.rpm
 
-RUN if [ "$(uname -m)" = "aarch64" ]; then dpdkRpmUrl=http://mirror.centos.org/altarch/7/extras/aarch64/Packages/dpdk-${dpdkVer}-${dpdkSubVer}.${rpmArch}.rpm; fi && rpm -i ${dpdkRpmUrl}
+RUN if [ "${rpmArch}" == "aarch64" ]; then dpdkRpmUrl=http://mirror.centos.org/altarch/7/extras/aarch64/Packages/dpdk-${dpdkVer}-${dpdkSubVer}.${rpmArch}.rpm; fi && rpm -i ${dpdkRpmUrl}
```

Comment:
Looks like uname -m  seems correct here. I might be wrong. A few lines above (line 37) we have a `rpmArch` definition to `x86_64`

Reply:
because rpmArch in the Makefile passed as "uname -m" see Line 34 in the Makefile below.